sass/style-ec.scss

```diff
@@ -4,11 +4,9 @@
 // TODO: The help form component is not available on ECL v3.
 //@use '@ecl/ec-component-form-help-block/ec-component-form-help-block';
 @use "./compositions/content_item";
-@use "./compositions/content_language_switcher";
```

Comment:
Why removing the language selector?

Reply:
Styles from this files aren't used anymore. Please check https://github.com/openeuropa/oe_theme/pull/1105/files#diff-2ff0353a221baabf5fe9662567a9de59fc9b3304137bb5b219a0cbd7ef25fbf3.
Language switcher isn't included in the page header in ECL v.3

Reply:
Alexandra insists that this file is requires but I don't understand why. Could someone else check this point.